Library/Homebrew/cmd/doctor.rb

```diff
@@ -428,6 +428,14 @@ def check_tmpdir_sticky_bit
   end
 end
 
+def check_supported_osx
+  if MacOS.version == "10.11" then <<-EOS.undent
+    Although Homebrew has added initial recognition of 10.11 it is unsupported.
```

Comment:
Would there be a comma appropriate here?:
Although Homebrew has added initial recognition of 10.11**,** it is unsupported.